index.html

```diff
@@ -136,6 +136,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+<!-- ________ Rahul Dev Sharma's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Rahul Dev Sharma</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com/rahulsust" target="_blank">Your handle</a>
+          </p>
+          <p class="about">I love playing cricket, code, cryptography. </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://frontendmasters.com/" target="_blank" title="Frontend Masters">
+                  Resource 1
+                </a>
+              </li>
+              <li>
+                <a href="freecodecamp.org" target="_blank" title="FreeCodeCamp">
+                  Resource 2
+                </a>
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="W3school">
+                  Resource 3
```

Comment:
Hello @rahulsust, thank you for contributing. Your code is formatted beautifully, but I noticed that you did not name the resource outside of just Resource 1,2, etc. I would suggest replacing that text with the name of the resource to make it more clear to those you are sharing your suggestions with. Make those changes, push your commit to this PR and we will get your card added. Take care 🚀

Reply:
Hi @lax0248917, thank for pointing out my mistake. I hope the problem is solved now. It will be pleasure if you inform for any furthur opportunity to contribute. 
cheers 🎉